scripts/dockerfiles/README.md

```diff
@@ -1,4 +1,92 @@
-# build the image
-docker build -t "binpash/pash:$distro-$version" .
-# push the image
-docker push binpash/pash:$distro-$version
+
+# PaSh Docker Support
+
+Docker files for [PaSh](https://github.com/binpash/pash) published on Dockerhub:
+
+* Ubuntu 18.04: [binpash/pash:ubuntu-18.04](https://hub.docker.com/r/binpash/pash)
+* Fedora 35 : [binpash/pash:fedora-35](https://hub.docker.com/r/binpash/pash)
+* Debian 10: [binpash/pash:debian-10](https://hub.docker.com/r/binpash/pash)
+
+## Run locally
+
+The following will pull and launch the latest image (tag: `pash:latest`) using the name `pash-play`
+
+```sh
+docker pull binpash/pash
+docker run --name pash-play -it pash
+```
+
+To restart after you exit, run `docker start -i pash-play`
+
+## From PaSh's Official Dockerfiles
+
+```sh
+git clone git@github.com:binpash/pash.git
+cd pash/scripts/docker
+docker build -t "pash:latest" .
+```
+
+## As a Dockerfile
```

Comment:
Do we need to duplicate the Dockerfile here? What if it changes? We could just point to it :)

Reply:
@angelhof Great idea!